crates/pixi_command_dispatcher/src/solve_pixi/source_metadata_collector.rs

```diff
@@ -172,15 +175,11 @@ impl SourceMetadataCollector {
 
         // Make sure that a package with the name defined in spec is available from the
         // backend.
-        if !source_metadata
-            .records
-            .iter()
-            .any(|record| record.package_record.name == name)
-        {
+        if source_metadata.records.is_empty() {
```

Comment:
What is the reason for this change?
If it is kept as is, I assume the comment above needs to be adapted